docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -0,0 +1,39 @@
+# Copyright IBM Corporation 2017.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk_container
+
+MAINTAINER Mark Stoodley <mstoodle@ca.ibm.com>
+
+# Install required OS tools
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    autoconf \
+    bison \
+    libffi-dev \
+    libnuma-dev \
+    g++-4.8 \
```

Comment:
Can you perhaps move all of this gcc stuff up to the main dockerfile. It currently fetches 4.8 already I think but we should be hard coding it anyway

Reply:
@gdams I agree, good idea to be explicit about the gcc/g++ versions used in the Docker builds for all OpenJDK.

Reply:
I could do if you would prefer. I was trying to isolate the changes that were needed for OpenJ9 to the -openj9 dockerfile. I didn't think OpenJDK required gcc 4.8 (or else your Dockerfile is broken already). By default, I believe you're getting gcc 5.something .